modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSQuickActions.testcase

```diff
@@ -104,17 +104,16 @@ definition {
 	}
 
 	@description = "LPS-155459: When clicking on the ellipsis and hovering over another row, multiple quick action menus are displayed"
-	@ignore = "true"
 	@priority = 3
 	test CanDisplayOnMultipleActiveRows {
 		task ("When click on ellipsis button 2nd row item") {
 			Click(
-				key_title = "Sample2",
+				key_itemName = "Sample3",
```

Comment:
```suggestion
				key_itemName = "Sample2",
```
Hey @andreluis2022 if we try this it might work, because from sample 3 to sample 2 I don't think it changes much. Could you try it and tell me if it works?